Editorial changes to MTF Examples (#41)
mulesoft/docs-mule-sdk: modules/ROOT/pages/mtf-using-test-classes.adoc

= Using Test Classes
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> When developing test cases, you may want to use Java classes located in `src/test/java`. Java classes can be used through the Java Module or through Dataweave.

Reviewer: ```suggestion
When developing test cases, you might want to use Java classes located in `src/test/java`. You can use Java classes through Java Module or through DataWeave.
```

== Using DataWeave

> To invoke static methods using DataWeave you must append the `java!` prefix to the package name of the class:

Reviewer: Are we putting `java!` at the beginning or the end?
"prefix" implies beginning; append implies ending.
If we say "add the `java!` prefix... we should be OK.
The alternative is to use "prepend" rather than "append."

> == Using the Java Module

Reviewer: Using Java Module

Because Java Module is a proper noun, it does not that the article "the" unless it is followed by a common noun as an object to the name.

> . Make sure the java module is your project:

Reviewer: ```suggestion
. Ensure that Java Module is your project:
```

</munit:test>
----

> The MUnit Extensions Maven Plugin automatically exports any test packages and test resources in your

Reviewer: ```suggestion
MUnit Extensions Maven Plugin automatically exports any test packages and test resources in your
```

project. If your module is already exporting a class located in `src/main/java`, then any test classes with the same package

> located in `src/test/java` won't be exported since there would be a conflict. To be able to export both classes, you must change the test classes to a different package.

Reviewer: ```suggestion
located in `src/test/java` won't be exported because this would cause a conflict. To be able to export both classes, you must change the test classes to a different package.
```
